src/_guides/libraries/writing-package-pages.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Writing package pages
+description: Learn how to write a good package page.
+---
+
+This document describes what you can do to construct a good package page on pub.dev. More specifically, this document provides several tips on how to write a better README.md file, as marked in the following image. If you need more information about other parts of the package page, refer to the links below. 
+
+{% asset libraries/package-page-sections.png alt="package page contains sections like package layout, flutter favorite, package scoring, verified publishers, pubspec file" %}
+
+1. About the [package layout ](https://dart.dev/tools/pub/package-layout)
+2. About the [Flutter Favorite program](https://flutter.dev/docs/development/packages-and-plugins/favorites)
+3. About the [package scoring system](https://pub.dev/help/scoring)
+4. About [verified publishers](https://dart.dev/tools/pub/verified-publishers)
+5. About the [pubspec file](https://dart.dev/tools/pub/pubspec)
+
+
+## Writing a good README is important
+
+Most package users who landed on your package on pub.dev will quickly scan your README to decide whether to try your package or not. A nice README will catch users’ attention and show that your package is worth trying.
+
+There are seven tips that you can follow to write a good README. 
```

Comment:
I'm curious if it is worth mentioning that it may be better to not include a Markdown title in the README as otherwise it will appear multiple times on pub.dev. Many of the Dart team's packages previously removed theirs as a result of this. An example of a package that still does include a Markdown title though is [dartdoc](https://pub.dev/packages/dartdoc). There is an [pub.dev issue](https://github.com/dart-lang/pub-dev/issues/3238) tracking automatic removal of these, but it seems to be on hold.

![image](https://user-images.githubusercontent.com/18372958/114804178-8182f380-9d66-11eb-9780-e7c1a49693e0.png)

Reply:
This is mentioned at the end of the first paragraph of tip 1, but it's easy to miss. I'll think about what to do...